servers/physics_3d/gjk_epa.cpp

```diff
@@ -1011,9 +1011,11 @@ bool gjk_epa_calculate_penetration(const GodotShape3D *p_shape_A, const Transfor
 	if (GjkEpa2::Penetration(p_shape_A, p_transform_A, p_margin_A, p_shape_B, p_transform_B, p_margin_B, p_transform_B.origin - p_transform_A.origin, res)) {
 		if (p_result_callback) {
 			if (p_swap) {
-				p_result_callback(res.witnesses[1], 0, res.witnesses[0], 0, p_userdata);
+				Vector3 normal = (res.witnesses[1] - res.witnesses[0]).normalized();
```

Comment:
res has a 'normal' field. Would this be better than calculating the witness point difference?

Reply:
Possibly?  I'm not at all familiar with the GJK code.  If it has already calculated a normal, all the better.

Reply:
Perhaps we can get @reduz to confirm this.

Reply:
Let's use `res.normal`.